### ISSUE TITLE
It started showing black screen when you start a video...

### ISSUE THREAD

**Issue by GiveNuts** [TheRealJoelmatic/RemoveAdblockThing#96]
![image](https://github.com/TheRealJoelmatic/RemoveAdblockThing/assets/149128122/a578d50c-288b-4ae7-a93a-1bdff3250da2)

When I start a video it shows the YouTube adblock thingy for a sec and after that the screen just goes black and you can't press anything on it.

**Comment by Shrey1517** [comment]
disable all other ad blockers or whitelist youtube in them

**Comment by WoeIsLee** [comment]
> disable all other ad blockers or whitelist youtube in them

I was able to turn them back on for the past two ish weeks or so, and then I disabled one earlier and was still able to keep watching with one working, but now none again.

**Comment by Domekologe** [comment]
I dont have any adblockers installed locally... there is an unwanted dns query blocker in our whole network which blocks requests to AD Services, which wont be disabled

**Comment by WoeIsLee** [comment]
> I dont have any adblockers installed locally... there is an unwanted dns query blocker in our whole network which blocks requests to AD Services, which wont be disabled

This is above my pay grade, dumb it down for me lol.

**Comment by Domekologe** [comment]
We use a system that handles all requests. This system automatically filters out all requests for advertising addresses. It is used so that you do not get any advertising even without an ad blocker.
Unfortunately, it also has the disadvantage that you can not turn off the ad blocker, if necessary.

**Comment by ddouglas87** [comment]
Same problem here.  When an ad blocker is on youtube shows a "Ad blockers violate YouTube's Terms of Service" pop up.  The ad blocker removes it and normally plays the video fine.  But the RemoveAdblockThing doesn't handle this situation appropriately resulting in a black screen. If RemoveAdblockThing is disabled one can watch Youtube videos fine.  It would be ideal if RemoveAdblockThing played nice with ad blockers, as some people can't remove ad blockers on their network and others don't want to have to be manually turn the RemoveAdblockThing script off and on all the time.

**Comment by WoeIsLee** [comment]
Just purge your ublock caches and update.
Been working for me since, but eventually its gonna stop working or someone is gonna find a more permanent fix.

**Comment by TheRealJoelmatic** [comment]
## Common Issues

**"Ad Blockers violate Youtube Terms Of Service" OR Just a black screen**

This happens when there is an adblock other than the script running. Common ways you may be blocking ads accidentally on Youtube is:

- **Adblock extension:** Disable or whitelist YouTube on your ad blocker extension.

- **Browser built-in adblock:** Some browsers have built-in ad blockers. Disable or configure them.

- **Other Tampermonkey scripts:** Scripts like YouTube enhancer may conflict. Disable them one by one to identify the issue.

- **Wi-Fi/DNS/Firewall adblock rules:** Check network settings for any ad-blocking configurations.

- **Windows hosts file rules:** Inspect the Windows hosts file (C:\Windows\System32\drivers\etc) for ad-blocking rules.
